### DISCUSSION TITLE
No support from github :(

### DISCUSSION THREAD

**Discussion by ghost** [community/community#147472] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Its been 3 months since my account got suspended/flaged for no reason and i have contacted github support in everyway i could like ticket,account reinstate,etc but still no reply from the support yet my account is still suspened i dont if im doing anything wrong here so if anyone can help me?

**Comment by milekv** [comment]
check your tickets – If you have a ticket number, reply to the existing thread instead of creating new ones. This might speed up the response.

use the appeal form – Submit the [Account Reinstatement Form](https://support.github.com/contact/account-reinstatement) if you haven't already.

reach out on Twitter – Contact [@GitHubSupport](https://twitter.com/githubsupport) with a brief explanation of your issue and include your ticket number.

contact Microsoft – Since GitHub is owned by Microsoft, try reaching out to Microsoft Support as well.

**Reply by ghost** [reply]
Githubsupport account doesnt exist its showing🥲

**Reply by ghost** [reply]
@milekv can you also say how to contact Microsoft? I tried via their site but there is no option for github

**Comment by MatinGhanbari** [comment]
Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)
